config/jobs/kubernetes/sig-gcp/sig-gcp-gke-config.yaml

```diff
@@ -442,33 +442,8 @@ periodics:
       - --timeout=150m
       image: gcr.io/k8s-testimages/kubekins-e2e:v20181005-fd9cfb8b0-master
 
-- interval: 1h
-  name: ci-kubernetes-e2e-gke-gci-ci-master
```

Comment:
why the removal of this job?

Reply:
this one is the same as the gke-gci job /shrug